v22.1/multiregion-overview.md

```diff
@@ -66,6 +66,32 @@ To show all of a database's regions, execute the [`SHOW REGIONS FROM DATABASE` s
 If the default survival goals and table localities meet your needs, there is nothing else you need to do once you have set a database's primary region.
 {{site.data.alerts.end}}
 
+## Super regions
+
+{% include common/experimental-warning.md %}
+
+<span class="version-tag">New in v22.1:</span> Super regions allow you to define a set of [database regions](#database-regions) such that the following [schema objects](schema-design-overview.html#database-schema-objects) will have all of their replicas stored _only_ in regions that are members of the super region:
+
+- [Regional tables](#regional-tables) whose home region is a member of the super region.
+- Any row of a [regional by row table](#regional-by-row-tables) whose [home region](set-locality.html#crdb_region) is a member of the super region.
+
+The primary use case for super regions is data domiciling. As mentioned above, data from [regional](#regional-tables) and [regional by row](#regional-by-row-tables) tables will be stored only in regions that are members of the super region. Further, if the super region contains 3 or more regions and if you use [`REGION` survival goals](#survive-region-failures), the data domiciled in the super region will remain available if you lose a region.
```

Comment:
s/Further/Furthermore (?)

Reply:
I just checked the macOS thesaurus and it appears they are synonyms

![Screen Shot 2022-05-17 at 10 13 59 AM](https://user-images.githubusercontent.com/1312291/168832132-02f8b198-916d-441c-9c0a-2aee1a3cfd6c.png)